resources/js/features/home/components/+sidebar/TopLinks/TopLinks.tsx

```diff
@@ -0,0 +1,87 @@
+import { useLaravelReactI18n } from 'laravel-react-i18n';
+import type { FC } from 'react';
+import {
+  LuBook,
+  LuCircleDot,
+  LuDownload,
+  LuHelpCircle,
+  LuNewspaper,
+  LuPodcast,
+} from 'react-icons/lu';
+import { PiMedalFill } from 'react-icons/pi';
+import { SiDiscord, SiPatreon } from 'react-icons/si';
+
+import { BaseButton, baseButtonVariants } from '@/common/components/+vendor/BaseButton';
+
+// TODO if config.services.discord.invite_id
+// TODO if config.services.patreon.user_id
+// TODO tracking
+
+export const TopLinks: FC = () => {
+  const { t } = useLaravelReactI18n();
+
+  return (
+    <div className="flex flex-col gap-2.5">
+      <a href={route('download.index')} className={baseButtonVariants({ size: 'sm' })}>
+        <LuDownload className="mr-2 h-4 w-4 text-sky-400" />
+        {t('Download Emulator')}
+      </a>
+
+      <div className="flex flex-col gap-2.5 sm:grid sm:grid-cols-2 lg:flex">
+        <a href="/globalRanking.php" className={baseButtonVariants({ size: 'sm' })}>
+          <LuCircleDot className="mr-2 h-4 w-4 text-amber-400" />
```

Comment:
> Uses similar iconography for points as introduced in 2758

Screenshots there have the medal for points and the circle for progress. Are these backwards?

Reply:
Good catch. I did indeed get these icons backwards. Fixed in latest.